Modules/@babylonjs/react-native-windows/windows/BabylonReactNative/EngineView.cpp

```diff
@@ -168,4 +170,22 @@ namespace winrt::BabylonReactNative::implementation {
     {
         BabylonNative::RenderView();
     }
+
+    void EngineView::UpdateProperties(IJSValueReader const& reader)
+    {
+        auto const& propertyMap = JSValueObject::ReadFrom(reader);
+
+        for (auto const& pair : propertyMap)
+        {
+            auto const& propertyName = pair.first;
+            auto const& propertyValue = pair.second;
+
+            if (propertyName == "isTransparent")
+            {
+                bool isTransparent = propertyValue.AsBoolean();
+                auto engineView = static_cast<winrt::Windows::UI::Xaml::Controls::SwapChainPanel>(*this);

```

Comment:
Nit: we already have `using namespace` directives for all of the `winrt::*` namespaces we use. You don't need to fully qualify these types.

Reply:
Fixed.